### DISCUSSION TITLE
[Fixed]Plugin no longer working in my Brave Browser

### DISCUSSION THREAD

**Discussion by koloman360** [Anarios/return-youtube-dislike#1004] [General]
Using Windows Brave V1.47.186 with RYD V3.0.0.12 . All Brave shields are disabled.

**Comment by Anarios** [comment]
Update to latest version (3.0.0.13)

**Reply by koloman360** [reply]
I didn't see a RYD update option in Brave so I assumed 3.0.0.12 was the current version... sorry.

**Reply by Anarios** [reply]
No problem, let me know if it fixed it for you.

**Reply by koloman360** [reply]
Working with V3.0.0.13. Thank you!

**Comment by valeriazagontova** [comment]
/

**Comment by Anarios** [comment]
Closing as resolved: the reporter confirmed this was fixed in 3.0.0.13. The current release is [v4.0.5](https://github.com/Anarios/return-youtube-dislike/releases/tag/v4.0.5).
